generate.js

```diff
@@ -45,7 +45,7 @@ const typescriptTemplate = {
   dir: 'app-ts',
   main: 'app.ts',
   scripts: {
-    test: 'npm run build:ts && tsc -p test/tsconfig.test.json && tap test/**/*.test.ts',
+    test: 'npm run build:ts && tap test/**/*.test.ts',
```

Comment:
```suggestion
    test: 'npm run build:ts && tap "test/**/*.test.ts"',
```
It is prefer to use quote here, because it can prevent the different of `globstar` behavior from `shell` and `platform`.
More discussion in [here](https://github.com/fastify/fastify-swagger/pull/351#discussion_r568819289)